### PR TITLE
Add random_page function

### PR DESCRIPTION
**Pull request by clareyong**

### CODE REVIEW

File: encyclopedia/views.py [hunk 1]
```diff
@@ -1,5 +1,6 @@
 from django.shortcuts import render, redirect
 import markdown2
+import random
```
Comment (kwinata) [inline encyclopedia/views.py:3]: It is customary to separate built-in import and packages import

Maybe we can change to 
```
import random

from django.shortcuts import render, redirect
import markdown2
```

File: encyclopedia/views.py [hunk 2]
```diff
@@ -40,3 +41,8 @@ def show_wiki(request, title):
         "entry_content": markdown2.markdown(util.get_entry(title)),
     }
     return render(request, "encyclopedia/entry.html", context)
+
+def random_page(request):
+    entries = util.list_entries()
+    random_page = entries[random.randint(0, len(entries)-1)]
```
Comment (kwinata) [inline encyclopedia/views.py:47]: You can also use `random.choice`, this one is simpler.

https://docs.python.org/3.8/library/random.html#random.choice

Example:
``` py

import random

mylist = ["apple", "banana", "cherry"]

print(random.choice(mylist)) 
```

Comment (clareyong) [reply encyclopedia/views.py]: Done

File: encyclopedia/views.py (hunk 1 above)
Comment (clareyong) [reply encyclopedia/views.py]: Done

**Review by kwinata — APPROVED** [review]

**Pull request merged**